modules/vaos/spec/serializers/v1/operation_outcome_serializer_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../../support/fixture_helper'
+
+describe VAOS::V1::OperationOutcomeSerializer do
+  describe 'json serialization' do
+    let(:resource_type) { :Organization }
+    let(:id) { '987654' }
+
+    context 'with a resource of Organization and a backend service exception' do
+      let(:issue) { Common::Exceptions::BackendServiceException.new('VAOS_502', source: 'Klass') }
+      let(:operation_outcome) { VAOS::V1::OperationOutcome.new(resource_type: resource_type, id: id, issue: issue) }
```

Comment:
These `let` statements are really the setup for the single example you follow with. Personally i think its clearer when the setup is in the example, but at least this is consistent in the spec.